engine/README.md

```diff
@@ -12,6 +12,17 @@
 - Now hopeit.engine package should be installed into your virtual/conda env linked to the source code in ./src
 - Then you can create your apps or plugins and run the server
 
+##### Install from Python Package Index
+
+- Install python 3.8
+- Create and activate a virtual environment (recommended)
+
+```
+pip install "hopeit.engine"
+pip intsall "hopiet.engine[web]"
+pip install "hopiet.engine[cli]"
```

Comment:
typo: hopiet

Suggestion: a single command can install all extras (please test before updating):
```
pip install "hopeit.engine[web,cli]"
```
Add explanation:

This will install core `hopeit.engine` lib with extras needed to run as a server and command line tools.

Reply:
@leosmerling-hopeit  applied the suggestions and made some changes, review and merge?